godot-codegen/src/generator/default_parameters.rs

```diff
@@ -97,6 +97,11 @@ pub fn make_function_definition_with_defaults(
     };
 
     let functions = quote! {
+    #[doc = "To set the default parameters, use [`"]
+    #[doc = stringify!(Self::#extended_fn_name)]
```

Comment:
This generates code containing a macro, which then again needs to be expanded by the compiler. Could you not create the entire doc-string at proc-macro time?

Then you could just have `#[doc = #link_to_ex_function]`.

Reply:
Any comment on this one? Why not generate the whole string at macro time and have a single `#[doc]` attribute? Would help readability and make it easier to change the text in the future than distributing it over 5 `#[doc = ...]` lines. 

And who knows, maybe it helps with the rustfmt issue, too 😉

Reply:
Ah sorry, I should've commented.  This is fixed in the version I pushed yesterday.  Unfortunately, no help with the rustfmt issue.

Reply:
fixed